python/setup.py

```diff
@@ -211,9 +211,9 @@ def get_option_from_sys_argv(option_str):
 
   # Keep this list of dependencies in sync with tox.ini.
   install_requires = ['six>=1.9', 'setuptools']
-  if sys.version_info <= (2,7):
-    install_requires.append('ordereddict')
+  if sys.version_info <= (2,6):
```

Comment:
Sorry for late review.
Version 2.6.9 is not <= (2,6).  sys.version_info < (2,7) make sense to me